### PR TITLE
3310 - Automatically fill in the tags and test name for the 'Add test…

### PR DESCRIPTION
**Pull request by florinbilt**
… data' module.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,7 +103,7 @@
     "html-loader": "4.2.0",
     "html-webpack-plugin": "5.5.4",
     "jest": "28.1.3",
-    "jest-environment-puppeteer": "9.0.1",
+    "jest-environment-puppeteer": "9.0.2",
```
Comment (beatrice-acasandrei) [inline package.json:106]: I think this shouldn't be part of this PR

**Comment by esanuandra** [comment]
We need to cover the following case:
1. Click on `Graphs` from the menu
2. Click on `Add test data`
3. 
Screenshot 2024-03-04 at 3 01 53 PM
